hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1501,6 +1501,87 @@
     }
 
     // KEYMGR (pre-verified IP) integration tests:
+    {
+      name: chip_keymgr_key_derivation
+      desc: '''Verify the keymgr advances to all states and generate identity / SW output.
+
+            - Backdoor load random value to OTP key, OTP device ID, Flash seeds and ROM degist
+              data.
+            - For HardwareRevisionSecret and LC diversification, use the constant values in design.
+            - Configure the keymgr and advance to `CreatorRootKey`.
+            - Generate identity / SW output for both Attestation CDI and Sealing CDI.
+            - KMAC should finish hashing successfully (not visible to SW) and return digest to
+              keymgr.
+            - Verify that the keymgr has received valid output from the KMAC.
+            - Advance to `OwnerIntermediateKey` and `OwnerRootKey` to repeat the above sequences.
+            - Generating identity / SW output in `OwnerRootKey` is optional as there is not
+              additional interaction between keymgr and other blocks.
+            - Advance to `Disabled` and verify keymgr enters the state successfully.
+            - Generate identity / SW output and ensure these are neither all 1s/0s nor any valid
+              key value, which proves secrets are wiped by entropy value from EDN.
+
+            - For each operation, wait for the interrupt `op_done` to be triggered and check CSR
+              `op_status` is `DONE_SUCCESS`.
+
+            - Note: there are 3 ways of calculating the digest data for comparison. Any of them is
```

Comment:
I think for the purpose of this test, it may be sufficient to just say 'matches the expected digest'. The actual mechanism of how the expected digest is computed is a testbench architecture detail.

Reply:
I think it's worth having some detail of implementation, otherwise, we might forget the discussion very soon